package.json

```diff
@@ -5,18 +5,18 @@
   "private": true,
   "scripts": {
     "postinstall": "husky install",
-    "boot": "yarn workspaces foreach -v -pA --topological-dev --no-private run build",
+    "boot": "yarn workspaces foreach -vv -pA --topological-dev --no-private run build",
```

Comment:
There are now two levels of verbosity. `-v` only adds the name of the workspace to the beginning of each log line, while `-vv` also adds the start/finish/timing log lines. The latter is default when running in the terminal, so I assume it's still needed when running in CI.